_layouts/default2025.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>{{ page.title }}</title>
+    <meta name="description" content="{{ page.description }}">
+    {% if page.stylesheet %}
+    <link rel="stylesheet" href="/assets/css/{{ page.stylesheet }}.css">
+    {% else %}
+    <link rel="stylesheet" href="/assets/css/styles.css">
+    {% endif %}
+    <link rel="icon" href="/assets/img/favicon.png" sizes="192x192">
+    <link rel="icon" href="/assets/img/favicon.ico" sizes="48x48">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+    {{ content }}
+    <div class="footerHolder">
+        <div class="container">
+            <footer>
+                <ul class="links">
+                    <li><a href="https://news.opensuse.org/">{{ locale.footer_news }}</a></li>
+                    <li><a href="https://doc.opensuse.org/">{{ locale.footer_docs }}</a></li>
+                    <li><a href="https://en.opensuse.org/Main_Page">{{ locale.footer_wiki }}</a></li>
+                    <li><a href="https://forums.opensuse.org/">{{ locale.footer_forums }}</a></li>
+                    <li><a href="https://software.opensuse.org/">{{ locale.footer_software }}</a></li>
+                    <li><a href="https://lists.opensuse.org/">{{ locale.mailing_lists }}</a></li>
+                    <li><a href="https://shop.opensuse.org/">{{ locale.footer_merch }}</a></li>
+                    <li><a href="https://planet.opensuse.org/">{{ locale.footer_planet }}</a></li>
+                    <li><a href="mailto:press@opensuse.org">{{ locale.footer_press_inquiries }}</a></li>
+                    <li><a href="https://en.opensuse.org/openSUSE:Artwork_brand#Trademark">{{ locale.footer_trademark }}</a></li>
+                    <li><a href="https://endof10.org/" target="_blank">End of 10</a></li>
+                </ul>
+                <div class="bottomBar">
+                    <div class="legalDisclaimer">
+                        <a href="https://www.opensuse.org">
+                            <img class="wordmark" alt="openSUSE wordmark" src="/assets/img/monochrome/wordmark.svg">
+                        </a>
+                        &copy; 2015-{{ site.time | date:"%Y" }} SUSE LLC. All Rights Reserved.
```

Comment:
SUSE LLC doesn't really own anything you make, does it? Make it openSUSE Contributors :D

Reply:
Done!